### DISCUSSION TITLE
Admin Page : Ability to sort by Districts

### DISCUSSION THREAD

**Discussion by jcookson-infoblox** [code4recovery/12-step-meeting-list#788] [General]
Am asking the question as well as making a request for a potential enhancement.
Currently the districts that utilize the TSML pieces, enter info in the "District" column in their import data.  This info populates and is then searchable in the front end UI.  However, the back end admin page still does not have the capability to search by all the districts that may have been there as result of association via this field.  Am requesting the capability to switch to districts, like the front end UI does.
Admin Page UI component,
<img width="841" alt="Screen Shot 2022-04-26 at 10 14 35 AM" src="https://user-images.githubusercontent.com/52213936/165358240-edbedfc7-c29b-4d9b-9425-196ec6f8ee02.png">
Region drop down component expanded,
<img width="842" alt="Screen Shot 2022-04-26 at 10 14 26 AM" src="https://user-images.githubusercontent.com/52213936/165358334-99978e23-4d25-49fe-a6f7-7b9e8d9d267f.png">

**Comment by brianw-area24** [comment]
We have a current issue, #370, about improving the searching in the back end admin page. At this point we don't have specific plans on what should be added there. Based on your question, I'll add districts as a possibility that we should consider...
